templates/python/.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
please add a CI workflow that runs pre-commit, similar to https://github.com/bakdata/streams-explorer/blob/7e657ddca4c7278affeeb31f74a7034f920c28b4/.github/workflows/ci.yml#L11

Reply:
I think that's done.